test/controllers/teams_controller_test.rb

```diff
@@ -305,9 +305,12 @@ def setup
     assert_select 'h3', 'Per User Statistics'
   end
 
-  test 'create team with location' do
-    team = create(:team, team_location: 'earth')
-    assert_equal team.team_location, 'earth'
+  test 'edit a teams location' do
```

Comment:
If this test is going to be dedicated to the team location and not just merged in with one of the other `:update` tests, then it should only set the team_location param for clarity. 

Also the original check you had in this test should not be deleted, it should just be moved into a model test.